.github/workflows/SyncToAtlas.yml

```diff
@@ -0,0 +1,54 @@
+name: Sync Master to Atlas Repos
+
+on: 
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  atlas:
+
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@master 
+    - name: Add O'Reilly remotes
+      run: |
+        git remote add oreilly git@git.atlas.oreilly.com:lenucksi/learningpath-introduction.git
+    - uses: fregante/setup-git-token@v1
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+    - name: enable SSH-Key
+      env: 
+        OREILLYKEY: ${{ secrets.OREILLYKEY }}
+      run: |
+        mkdir -p $HOME/.ssh
+        #ls -al ~/.ssh
+        echo -e "$OREILLYKEY" > ~/.ssh/id_rsa
+        chmod 600 ~/.ssh/id_rsa
+        eval `ssh-agent -s`
```

Comment:
Just curious - why is `eval` needed here instead of just running `ssh-agent -s` directly?

Reply:
`ssh-agent` outputs some env variables when it runs. if you wrap it in an `eval`, it activates those and makes the agent available to subsequent calls.
https://linux.die.net/man/1/ssh-agent (look under _There are two main ways to get an agent set up_)

Reply:
Got it.